decidim-core/app/presenters/decidim/participatory_process_stats_presenter.rb

```diff
@@ -35,10 +35,10 @@ def feature_stats(conditions)
     end
 
     def render_stats_data(feature_manifest, name, data, index)
-        safe_join([
-                    index == 0 ? feature_manifest_icon(feature_manifest) : " /&nbsp".html_safe,
-                    content_tag(:span, "#{number_with_delimiter(data)} " + I18n.t("#{name}", scope: "decidim.participatory_processes.statistics"), class: "#{name} process_stats-text")
-                  ])
+      safe_join([
+                  index == 0 ? feature_manifest_icon(feature_manifest) : " /&nbsp".html_safe,
+                  content_tag(:span, "#{number_with_delimiter(data)} " + I18n.t(name.to_s, scope: "decidim.participatory_processes.statistics"), class: "#{name} process_stats-text")
```

Comment:
You shouldn't need the `to_s` part, since it will only be a string or a symbol and `I18n.t` should handle both cases 😕